ST_structs.h

```diff
@@ -284,21 +285,12 @@ struct globals_st {
   RealF plotsize,   /* size of plot in square meters */
         gsppt_prop, /* proportion of ppt during growing season*/
         tempparm[2][3]; /* three parms for Warm/Cool growth mod*/
-  IntUS runModelYears,  /* number of years to run the model*/
-      Max_Age,        /* oldest plant; same as runModelYears for now */
+  IntUS Max_Age,        /* oldest plant; same as runModelYears for now */
```

Comment:
@kpalmqui It is used when outputting mortality information. I'm not sure why it is used as opposed to Globals->runModelYears since they are the same, but my impression is that someone was going to set Max_Age to species-specific values but never got around to it.

Reply:
@chaukap ok -thanks!